src/ResourceManager/Network/Commands.Network/AzureRM.Network.psd1

```diff
@@ -335,6 +335,7 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Remove-AzureRmServiceEndpointPolicyDefinition', 
                'Get-AzureRmServiceEndpointPolicyDefinition',
                'Set-AzureRmServiceEndpointPolicyDefinition',
+               'Add-AzureRmServiceEndpointPolicyDefinition',
```

Comment:
is this change present in the netcore psd1 as well? if not, do kindly add it there as well...

Reply:
Fixed.